### PR TITLE
HELP-15581: Log when not config menu option was selected after 3 tries

### PR DESCRIPTION
**Pull request by harenson**
4.3: https://github.com/2600hz/kazoo/pull/6604

### CODE REVIEW

File: src/module/cf_voicemail.erl [hunk 1]
```diff
@@ -1282,24 +1283,26 @@ config_menu(Box, Call) ->
 config_menu(#mailbox{interdigit_timeout=Interdigit}=Box
            ,Call
            ,Loop
-           ) when Loop < 4 ->
-    lager:info("playing mailbox configuration menu"),
+           ) when Loop =< ?MAX_CONFIG_MENU_LOOPS ->
```
Comment (icehess) [inline src/module/cf_voicemail.erl:1286]: Is there a reason that you're changing this to 3 retries?

**Review by icehess — CHANGES_REQUESTED** [review]

File: src/module/cf_voicemail.erl (hunk 1 above)
Comment (harenson) [reply src/module/cf_voicemail.erl:1286]: It starts from 0 (zero) so it loops actually 4 (0-3) times before throwing the error message. I think it was just to make it look the same as the `MAX_INVALID_PIN_LOOPS` macro. Also, it is the same to say `X < 4` than saying `X =< 3` ;)

To be honest I don't recall at the moment why I did it. I can change it back to `Loop < 4` if you want me to do it.

Comment (icehess) [reply src/module/cf_voicemail.erl:1286]: It starts at 0?

```
1285:config_menu(Box, Call) ->
1286:    config_menu(Box, Call, 1).
```

Comment (harenson) [reply src/module/cf_voicemail.erl:1286]: @icehess You are correct, but it is still looping 3 times as it was before that change. And to answer your question IIRC it was changed to 3 and set to `MAX_CONFIG_MENU_LOOPS` macro just to make its use clear on `config_menu/3 when Loop > ?MAX_CONFIG_MENU_LOOPS` function clause.

**Comment by harenson** [comment]
Thanks for merging this one. May you also merge the 4.3 backport, please? https://github.com/2600hz/kazoo/pull/6604

**Pull request merged**